winch/codegen/src/stack.rs

```diff
@@ -275,6 +283,11 @@ impl Stack {
         self.inner.range(partition..)
     }
 
+    /// Returns an iterator of the stack in top-most to bottom-most order.
+    pub fn iter(&self) -> impl Iterator<Item = &Val> + '_ {
```

Comment:
I looked at using `peekn` but the iterator it returns is in bottom-most to top-most order and the return type is not a trait that allows me to reverse the order of the entries so I can ignore the top-most element of the stack.